stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -78,8 +79,11 @@ public fun ChannelInfo(
         ?.role
         ?.let { it == "admin" || it == "owner" }
         ?: false
-
-    val otherMembers = channelMembers.filter { it.user.id != currentUser?.id }
+    val membersToDisplay = if (selectedChannel.isOneToOne(currentUser)) {
+        channelMembers.filter { it.user.id != currentUser?.id }
+    } else {
+        channelMembers
+    }
```

Comment:
Displaying the current user in the list of channel members like other SDKs. Also, makes sense as we, for example, have a caption that the channel contains 3 members, and we actually have 3 use avatars below (including the current user).

Reply:
Makes total sense, nicely done!